src/devices/tpms_ford.c

```diff
@@ -52,12 +57,21 @@ static int tpms_ford_decode(r_device *decoder, bitbuffer_t *bitbuffer, unsigned
 
     code = b[4]<<16 | b[5]<<8 | b[6];
     sprintf(code_str, "%06x", code);
+    
+    pressure = 0.3 + b[4] * 0.25;
```

Comment:
is this perhaps `(b[4] + 1) * 0.25f` ?

Reply:
No. Tested: when 00 forscan read from BdyCM value 2.1kPa. This is not correct because no pressure and outbut must be 0. And there are different step and start value.